CMakeLists.txt

```diff
@@ -7,11 +7,13 @@ add_compile_options(-W)
 add_compile_options(-Wall)
 add_compile_options(-Wno-restrict) # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=104336
 add_compile_options(-Wno-stringop-overread) # ditto
-add_compile_options(-O3)
+#add_compile_options(-O3)
```

Comment:
Is `-O3` causing problems, or was this for debugging? In general it might be handy if we can keep this sort of thing separated out from feature commits.

Reply:
Ah whoops yes sorry this was just for debugging